content/doc/developer/plugin-development/dependency-management.adoc

```diff
@@ -4,18 +4,22 @@ title: Dependency Management
 references:
 - url: https://github.com/jenkinsci/plugin-pom/
   title: Plugin parent POM 4.0 and up
+////
+Should we be linking to the generic BOM or encouraging people to use the BOM
```

Comment:
this one is correct

Reply:
Thanks -- I deleted the question.

Reply:
Thanks!  I removed the question from the source.